nengo_spa/tests/test_semantic_pointer.py

```diff
@@ -307,22 +341,41 @@ def test_invalid_algebra():
     SemanticPointer(next(gen), algebra=HrrAlgebra())
 
 
-def test_identity(algebra):
-    assert np.allclose(Identity(64, algebra=algebra).v, algebra.identity_element(64))
+@pytest.mark.filterwarnings("ignore:.*:DeprecationWarning")
+@pytest.mark.parametrize("sidedness", ElementSidedness)
```

Comment:
Could you help me understand why `ElementSideness` is a `parametrize` parameter? It seems like a constant that could be just defined in the function?

Reply:
Depending on the `algebra` (which is also parametrized), there might only be identities for specific sides. Left and right identity might also not be the same in the general case (though currently they are for `HrrAlgebra` and `VtbAlgebra` does only have a right identity).

Reply:
Oh, I didn't understand that `ElementSidedness` meant it would iterate through each value of the `Enum` as a parameter.